### DISCUSSION TITLE
How to set public login page info

### DISCUSSION THREAD

**Discussion by botzill** [directus/directus#3009] [Archive]
I saw in the demo you have:

<img width="1259" alt="Screen Shot 2020-11-07 at 18 10 02" src="https://user-images.githubusercontent.com/4686040/98446253-de79de00-2124-11eb-8a6a-b57c03fb483f.png">

Is possible to easily configure that?

**Comment by benhaynes** [comment]
Yes, just configure the public fields within settings. 

I believe the names are: `project_logo`, `public_background`, `public_foreground`, and `public_note`
